### ISSUE TITLE
The support for masking is not implemented for Bidirectional layers yet

### ISSUE THREAD

**Issue by BKHMSI** [tensorflow/tfjs#2279]
#### TensorFlow.js version
1.2.11
#### Browser version
Chrome 78.0

#### Describe the problem or feature request
I converted a keras model that contains a Bidirectional LSTM layer with `mask_zero=True` in its previous Embedding layer. I converted the model using the [tensorflowjs_convertor](https://www.tensorflow.org/js/tutorials/conversion/import_keras), and it loads as expected (outputs the model summary correctly), but when I run `model.predict(input)` it generates this error: "_The support for masking is not implemented for Bidirectional layers yet_". 

I don't even want masking in this case because I just want to predict not train using tensorflowjs!

**Update**

I solved the issue by creating an empty model with the  `mask_zero=False` in the Embedding layer, then loading my previous weights into the model. 

I just want to note that tfjs should handle such cases automatically when the model is only being used for predictions (i.e. such an error message is valid if I attempted to train the model when masking is not yet implemented for Bidirectional layers)!
